app/assets/stylesheets/custom.scss

```diff
@@ -291,3 +291,33 @@ ul, menu, dir {
 .filter {
   text-align: center;
 }
+
+
+// /* =========================
+//
+//      USER LISTINGS INDEX PAGE
+//
+//    =========================*/
+
+#img_container {
+    position:relative;
+    display:inline-block;
+    text-align:center;
+}
+
+.edit-button {
+    position:absolute;
+    bottom:10px;
+    right:100px;
+    width:100px;
+    height:30px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Colon after property should be followed by one space